sklearn/feature_selection/tests/test_rfe.py

```diff
@@ -491,3 +492,11 @@ def test_multioutput(ClsRFE):
     clf = RandomForestClassifier(n_estimators=5)
     rfe_test = ClsRFE(clf)
     rfe_test.fit(X, y)
+
+
+# TODO: Remove in version 0.26
+def test_deprecated_estimator_type():
+    # Assert that deprecated _estimator_type warns FutureWarning
+    rfe = RFE(SVC())
+    with pytest.warns(FutureWarning):
+        hasattr(rfe, "_estimator_type")
```

Comment:
We should also check warning messages  (here and everywhere else)

Also nit but I think `getattr` would be more obvious

Reply:
You are absolutely right, I forgot to add the `match` message.